### PR TITLE
Deno port

### PR DESCRIPTION
**Pull request by K0IN**
Porting backend to deno

### CODE REVIEW

**Comment by K0IN** [comment]
TLDR:
I wanted to write it in deno in the first place
There was no production grade tool to run cf workers
Testing locally in miniflare was sometimes challenging

Btw there is a [branch](https://github.com/K0IN/Notify/tree/cloudflare-legacy) that still works with Cloudflare, maybe in the future i will support cf again :).

Long Story:
Actually I wanted to write this app in Deno from the start, but at the time Deno did not support the full web crypto standard.
This project started on Sep 11, 2021, Deno first (not full) web crypto release was in [1.12](https://deno.com/blog/v1.12) on July 13, 2021 and the full functionality since [1.18](https://deno.com/blog/v1.18) (January 20, 2022).

Running locally, was one of my main objective for this tool.

But i always had some issues with running Cloudflare functions locally, sure there is Minfilare and workerd (And denoflare - which had the same web crypto issues stated above).
But both come with some issues:

Miniflare - is not (or atleast i could not finde any evidence) production ready, as i understand it is just for testing "Miniflare is a simulator for developing and testing"(https://miniflare.dev/), so i do not deem it secure nor reliable. I must say that it was on my part that i miss that, i had a instance running from like early development until the deno switch without any downtime or issues tho.

Workerd - is a proper way to host cf functions, but at time of thinking to port this workerd did not support kv storage (or i was to stupid to find out how).


I also had some issues with miniflare, which I just found annoying and time-wasting (after all, this is just a weekend project), i felt that debugging never was great, and my integration tests were a bit hacky.

**Comment by MiguelRipoll23** [comment]
@K0IN you should release your web push port as a deno module.

the npm one doesn’t work and webpush-webcrytp apparently does not play well with WebKit.

**Comment by K0IN** [comment]
Yes, I thought of that, but keep in mind my code has not been audited, which is bad since it uses a lot of cryptography.
So releasing this without an audit is a bit rectless IMHO. 
I review my code, to my best of my knowledge, but can never guaranty that i don't make mistakes.
In a “low stakes” app like notify I'm willing to take this risk, but i don't want someone else, to take the risk (maybe unknowingly).

So yeah, possibly in the future but certainly not now.
